components/src/core/components/Dialog/Dialog.vue

```diff
@@ -69,13 +69,18 @@ export default defineComponent({
       type: Boolean,
       default: true,
     },
+    dialogSizeClass: {
+      type: String,
+      default: 'modal-md',
```

Comment:
Can we define type for this? so we can have validation for the size class. 
https://github.com/orangehrm/oxd/blob/ent/components/src/core/components/Input/Select/SelectInput.vue#L101